specification/datalake-analytics/data-plane/Microsoft.DataLakeAnalytics/2017-09-01-preview/job.json

```diff
@@ -0,0 +1,1974 @@
+        "details": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the details of the audit log."
+        }
+      },
+      "description": "The Data Lake Analytics job state audit records for tracking the lifecycle of a job."
+    },
+    "ScopeJobResource": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "the name of the resource."
+        },
+        "path": {
+          "type": "string",
+          "description": "the path to the resource."
+        }
+      },
+      "description": "The Scope job resources. (Only for use internally with Scope job type.)"
+    },
+    "JobResource": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "the name of the resource."
+        },
+        "resourcePath": {
+          "type": "string",
+          "description": "the path to the resource."
+        },
+        "type": {
+          "type": "string",
+          "description": "the job resource type.",
+          "enum": [
+            "VertexResource",
+            "JobManagerResource",
+            "StatisticsResource",
+            "VertexResourceInUserFolder",
+            "JobManagerResourceInUserFolder",
+            "StatisticsResourceInUserFolder"
+          ],
+          "x-ms-enum": {
+            "name": "JobResourceType",
+            "modelAsString": false
+          }
+        }
+      },
+      "description": "The Data Lake Analytics job resources."
+    },
+    "USqlJobProperties": {
+      "x-ms-discriminator-value": "USql",
+      "allOf": [
+        {
+          "$ref": "#/definitions/JobProperties"
+        }
+      ],
+      "properties": {
+        "resources": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/JobResource"
+          },
+          "description": "the list of resources that are required by the job"
+        },
+        "statistics": {
+          "$ref": "#/definitions/JobStatistics",
+          "description": "the job specific statistics."
+        },
+        "debugData": {
+          "$ref": "#/definitions/JobDataPath",
+          "description": "the job specific debug data locations."
+        },
+        "diagnostics": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Diagnostics"
+          },
+          "description": "the diagnostics for the job."
+        },
+        "algebraFilePath": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the algebra file path after the job has completed"
+        },
+        "totalCompilationTime": {
+          "type": "string",
+          "format": "duration",
+          "readOnly": true,
+          "description": "the total time this job spent compiling. This value should not be set by the user and will be ignored if it is."
+        },
+        "totalPausedTime": {
```

Comment:
Renaming is a breaking change - please be aware of that. Also you'll need to update PowerShell & CLI cmdlets if you have any.

Reply:
Thank you for noting this.  I should have been clear in my notes, but this is actually a bugfix.  Yes it would be a breaking change, but the service returns this as TotalPausedTime.  I don't expect this property to be too popular among users.  Will keep an eye out for this when I update PS and CLI.